packages/cmf/src/expressions/oneOf.js

```diff
@@ -0,0 +1,13 @@
+import _get from 'lodash/get';
```

Comment:
Wouldn't it be good to keep cherry picked parts of lodash the same way across all files ? `get` instead of `_get`